molvis/molvispython/python/molviscommon.py

```diff
@@ -31,6 +31,8 @@
 import ivwdataframe as df
 import ivwmolvis
 
+import ivwmolvis.atomicelement as atomicelement
```

Comment:
```suggestion
from ivwmolvis import atomicelement
```

Reply:
it's the same. Experimented first with a different name instead of `atomicelement`.